front/components/poke/plans/form.tsx

```diff
@@ -60,6 +61,7 @@ export const fromPlanType = (plan: PlanType): EditingPlanType => {
     dataSourcesDocumentsSizeMb: plan.limits.dataSources.documents.sizeMb,
     maxUsers: plan.limits.users.maxUsers,
     billingType: plan.billingType,
+    trialPeriodDays: plan.trialPeriodDays ?? 0,
```

Comment:
So if we edit we move it from null to 0 right?

Reply:
Just in the form. We still use `null` in DB

Reply:
Yes. Actually I can't really decide what is best ? Do we want to keep `null` or use 0 everywhere (and by default) ?

Reply:
I feel like there's not really a reason to use nulls. I'll just go `0` everywhere (and have non-nullable, 0 as default in schema)

Reply:
I'm fine with either or but I think we want consistency. Aka all null if non zero or all zero :+1:

Reply:
We don't send the `0` to stripe. We replace by `undefined`. We want to have this logic before sending to strip regardless of nullability, so IMO having 0 is just better (consistency is guaranteed, less ambiguous)